mrpt_rbpf_slam/include/mrpt_rbpf_slam/mrpt_rbpf_slam.h

```diff
@@ -3,9 +3,7 @@
  *  Author: Vladislav Tananaev
  */
 
-#ifndef MRPT_RBPF_SLAM_H
-#define MRPT_RBPF_SLAM_H
-
+#pragma once
```

Comment:
Agreed.

Give this a try ;-)
https://github.com/marcusmueller/include-guard-convert

Reply:
Thanks! I will do a separate PR with that!